### DISCUSSION TITLE
Add more sign-in providers w/ Firebase Authentication

### DISCUSSION THREAD

**Discussion by sreedhara-aneesh** [csc510-fall22-p2-g33/FlashCards#23] [Ideas]
Users should be able to sign in with external accounts (Google, Apple, Microsoft, etc) rather than being forced to create a new account solely for this application. Firebase Authentication allows for simple, secure authorization flows. The ability to easily add new authentication flows/providers as we see fit over time will benefit the scalability of this application.

**Comment by sreedhara-aneesh** [comment]
Being worked on, will be complete soon (see #29).

**Reply by mkudith** [reply]
Is this a duplicate of #29?
